sdks/java/io/debezium/expansion-service/build.gradle

```diff
@@ -20,6 +20,17 @@ apply plugin: 'org.apache.beam.module'
 apply plugin: 'application'
 mainClassName = "org.apache.beam.sdk.expansion.service.ExpansionService"
 
+// The order is intended here - Debezium 3 requires Java 17, or later
```

Comment:
same, use `requireJavaVersion = JavaVersion.VERSION_17` and remove the version logics here and at the bottom of the file